crates/languages/src/python.rs

```diff
@@ -259,8 +264,8 @@ async fn get_cached_server_binary(
 
 pub(crate) struct PythonContextProvider;
 
-const PYTHON_UNITTEST_TARGET_TASK_VARIABLE: VariableName =
-    VariableName::Custom(Cow::Borrowed("PYTHON_UNITTEST_TARGET"));
+const PYTHON_TEST_TARGET_TASK_VARIABLE: VariableName =
```

Comment:
On one hand this is a breaking change, but on the other I don't think we have this variable documented anywhere. We should be good.